recipes/_user.rb

```diff
@@ -95,14 +96,12 @@
   else
     directory build_user_home do
       owner node['omnibus']['build_user']
-      group node['omnibus']['build_user_group']
       action :create
     end
   end
 else
   directory build_user_home do
     owner node['omnibus']['build_user']
-    group node['omnibus']['build_user_group']
```

Comment:
Instead of removing this line, I think it probably makes sense to make the group property for these directory resources conditional on windows platform, e.g.:

```
group node['omnibus']['build_user_group'] unless windows?
```

I have similar changes in our fork of this cookbook, which I will gladly open a PR for if that will help to move this change along.